powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/UnsupportApplicationVersionRequest.java

```diff
@@ -28,7 +30,12 @@
 @Data
 public class UnsupportApplicationVersionRequest {
 
+    @Schema(description = "The identifier of the application")
+    @NotBlank(message = "Application ID must not be empty when unsupporting application version")
```

Comment:
Suggesting different wording.
```suggestion
    @NotBlank(message = "Application ID must not be empty when marking an application version as unsupported")
```